tests/dummy/app/templates/typography.hbs

```diff
@@ -1,6 +1,6 @@
 {{page-toolbar pageTitle="Typography" isDemo=false}}
 
-{{#paper-content class="md-padding"}}
+{{#paper-content class="md-padding" tabindex="0"}}
```

Comment:
This looks good to me. Maybe @miguelcobain can take a peek. I just added `theme.hbs`, so that would need a `tabindex` too.